scripts/pretty-printers/gdb/pretty_printers.py

```diff
@@ -1,6 +1,43 @@
 import gdb
 
 
+def deconstruct_dstring(val):
```

Comment:
:pick: Probably would have been easier to review if the refactoring of printing dstring was separate to the introduction of irep printing

Reply:
That might be slightly overkill - the changes can be read top-to-bottom - only the last hunk is not related to this refactoring.